code/clue.js

```diff
@@ -1,59 +1,190 @@
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 82
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const poison = {
+  name: 'poison',
+  weight: 8
+}
+
+const axe = {
+  name: 'axe',
+  weight: 56
+}
+const bat = {
+  name: 'bat',
+  weight: 38
+}
+const trophy = {
+  name: 'trophy',
+  weight: 47
+}
+const pistol = {
+  name: 'pistol',
+  weight: 32
+}
 
 const suspects = [
   mrGreen,
+  prPlum,
+  msScarlett,
+  mrsPeacock,
+  clMustard,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
+
+const rooms = [
+  'dining room',
+  'conservatory',
+  'kitchen',
+  'study',
+  'library',
+  'billiard room',
+  'lounge',
+  'ballroom',
+  'hall',
+  'spa',
+  'living room',
+  'observatory',
+  'theater',
+  'guest house',
+  'patio'
+]
 
-const rooms = []
+console.log(weapons[0])
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+let mystery = {}
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
+
   mystery.killer = randomSelector(suspects)
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
+  document.getElementById('killerAge').innerHTML = mystery.killer.age
+  document.getElementById('killerOccupation').innerHTML = mystery.killer.occupation
+  document.getElementById('killerImage').src = mystery.killer.image
   document.getElementById(
     'killerName'
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
 }
 
-// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+document.getElementById('killerCard').onclick = pickKiller
+
+const pickWeapon = () => {
+
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponWeight').innerHTML = mystery.weapon.weight
+  document.getElementById(
+    'weaponName'
+  ).innerHTML = `${mystery.weapon.name}`
+}
+
+document.getElementById('weaponCard').onclick = pickWeapon
+
+console.log(mystery)
+
+const pickRoom = () => {
+
+  mystery.room = randomSelector(rooms)
+  document.getElementById(
+    'roomName'
+  ).innerHTML = `${mystery.room}`
+}
+
+document.getElementById('roomCard').onclick = pickRoom
+
+console.log(mystery)
+
+const revealMystery = () => {
```

Comment:
If I click `Reveal the crime` in the browser before choosing anything, I see an error in the console.  Would be cool if the user got a message about why nothing happens